apps/src/music/views/HeaderButtons.tsx

```diff
@@ -141,6 +142,8 @@ const HeaderButtons: React.FunctionComponent<HeaderButtonsProps> = ({
     }
   }, [dialogControl, skipUrl]);
 
+  const settings = useBlocklySettings();
```

Comment:
This had to be moved to a constant so that we weren't calling the hook conditionally.